clients/python/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.mypy]
+# Global options
+check_untyped_defs = true
+disallow_any_generics = false
+# disallow_untyped_defs: if True, it enforces things like `def __init__(self) -> CLASSNAME` or `def test_() -> None` which does not worth the effort
+disallow_untyped_defs = false
+follow_imports = "silent"
+# ignore_missing_imports: removes all the missing imports stuff from external libraries which is annoying to the least
+ignore_missing_imports = true
```

Comment:
This one will bite you back heavily! That is precisely why we had such a mess in simcore